tutorials/applications/Shadow estimation for VQS.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    dist = np.zeros(5)
</pre><p>For conciseness, maybe create a shadow_sizes list =[100, 1000, 2000, 5000, 10000]. Re-use it in the plt.plot line below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>